include/tacho_motor_class.h

```diff
@@ -195,6 +196,7 @@ struct tacho_motor_ops {
 	int (*set_hold_Kd)(void *context, int k);
 };
 
+extern int tacho_motor_get_state(struct tacho_motor_device *tm);
```

Comment:
I don't see this used anywhere outside of `tacho_motor_class.c`. Why do we need it in the header file?